include/tvm/runtime/packed_func.h

```diff
@@ -640,20 +640,21 @@ inline const char* TypeCode2Str(int type_code) {
   }
 }
 
+#ifndef _LIBCPP_SGX_NO_IOSTREAMS
 inline std::ostream& operator<<(std::ostream& os, TVMType t) {  // NOLINT(*)
-  os << TypeCode2Str(t.code);
-  if (t.code == kHandle) return os;
-  os << static_cast<int>(t.bits);
-  if (t.lanes != 1) {
-    os << 'x' << static_cast<int>(t.lanes);
-  }
-  return os;
+  return os << TVMType2String(t);
```

Comment:
Keep the original content of the serialization, even if it is a bit duplicating, this one can be more efficient

Reply:
keep original variant of TVMType2String here